### ISSUE TITLE
Doing `require 'backports'` is deprecated

### ISSUE THREAD

**Issue by CineEncoder** [jordansissel/fpm#1723]
Hi,

Please tell me why this error occurs?
```
[helg@Houston Downloads]$ fpm -s python -t rpm CineEncoder-2.1/setup.py
Doing `require 'backports'` is deprecated and will not load any backport in the next major release.
Require just the needed backports instead, or 'backports/latest'.
/usr/lib/ruby/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
Need executable 'rpmbuild' to convert python to rpm {:level=>:error}
```

**Comment by luis-lavaire** [comment]
I can confirm the same error. I installed it in Arch Linux a couple minutes ago.

**Comment by CineEncoder** [comment]
I have the same problem on both Manjaro and Ubuntu 20.

**Comment by ABitMoreDepth** [comment]
@CineEncoder are you asking specifically about the `require 'backports'` notice, or are you asking why an attempt to build something failed?

The final line in your provided output is the most valuable snippet: `Need executable 'rpmbuild' to convert python to rpm {:level=>:error}`

On redhat based systems (like centos, fedora etc.), the tool to create an RPM, `rpmbuild` will be available by default, however, neither Manjaro or Ubuntu will by default have that tool available.  On Ubuntu, I was able to start creating rpm files after running `apt install rpm`, which makes the relevant tool that FPM requires available.

I'm not very familiar with Manjaro, but it looks like [this](https://www.archlinux.org/packages/community/x86_64/rpm-tools/) package should achieve a similar result.

**Comment by brucelle** [comment]
Good afternoon.

I'm getting the following errors. Any assistance will be greatly appreciated. Thank you.


Doing `require 'backports'` is deprecated and will not load any backport in the next major release.

Require just the needed backports instead, or 'backports/latest'.

**_Process failed: rpmbuild failed (exit code 1). Full command was:["rpmbuild", "-bb", "--target", "noarch", "--define", "buildroot /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/BUILD", "--define", "_topdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_sourcedir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_rpmdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/RPMS", "--define", "_tmppath /tmp", "/tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/SPECS/root.spec"] {:level=>:error}_**

**Comment by ariz-mendi** [comment]
Good afternoon.

I'm getting the following errors. Any assistance will be greatly appreciated. Thank you.

Doing require 'backports' is deprecated and will not load any backport in the next major release.

Require just the needed backports instead, or 'backports/latest'.

_Process failed: rpmbuild failed (exit code 1). Full command was:["rpmbuild", "-bb", "--target", "noarch", "--define", "buildroot /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/BUILD", "--define", "_topdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_sourcedir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_rpmdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/RPMS", "--define", "tmppath /tmp", "/tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/SPECS/root.spec"] {:level=>:error}

**Comment by ariz-mendi** [comment]
Good afternoon.

I'm getting the following errors. Any assistance will be greatly appreciated. Thank you.

Doing require 'backports' is deprecated and will not load any backport in the next major release.

Require just the needed backports instead, or 'backports/latest'.

_Process failed: rpmbuild failed (exit code 1). Full command was:["rpmbuild", "-bb", "--target", "noarch", "--define", "buildroot /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/BUILD", "--define", "_topdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_sourcedir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_rpmdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/RPMS", "--define", "tmppath /tmp", "/tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/SPECS/root.spec"] {:level=>:error}

**Comment by ariz-mendi** [comment]
Good afternoon.

I'm getting the following errors. Any assistance will be greatly appreciated. Thank you.

Doing require 'backports' is deprecated and will not load any backport in the next major release.

Require just the needed backports instead, or 'backports/latest'.

_Process failed: rpmbuild failed (exit code 1). Full command was:["rpmbuild", "-bb", "--target", "noarch", "--define", "buildroot /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/BUILD", "--define", "_topdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_sourcedir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_rpmdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/RPMS", "--define", "tmppath /tmp", "/tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/SPECS/root.spec"] {:level=>:error}

**Comment by mendi70** [comment]
Good afternoon.

I'm getting the following errors. Any assistance will be greatly appreciated. Thank you.

Doing require 'backports' is deprecated and will not load any backport in the next major release.

Require just the needed backports instead, or 'backports/latest'.

_Process failed: rpmbuild failed (exit code 1). Full command was:["rpmbuild", "-bb", "--target", "noarch", "--define", "buildroot /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/BUILD", "--define", "_topdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_sourcedir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6", "--define", "_rpmdir /tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/RPMS", "--define", "tmppath /tmp", "/tmp/package-rpm-build-e16b089064454a9323f0945d9444a28f11e36865a20d09b75db1a16b02a6/SPECS/root.spec"] {:level=>:error}

**Comment by BartKoppers** [comment]
Merge with #1716

**Comment by jordansissel** [comment]
This was fixed by #1727 and first released in fpm 1.12.0 (2021/01/19)
